src/masternodes/rpc_accounts.cpp

```diff
@@ -2147,15 +2146,17 @@ UniValue getburninfo(const JSONRPCRequest& request) {
     g.WaitForCompletion();
 
     for (const auto &r : resultsPool.GetContainer()) {
-        totalResult->burntDFI += r.burntDFI;
-        totalResult->burntFee += r.burntFee;
-        totalResult->auctionFee += r.auctionFee;
-        totalResult->burntTokens.AddBalances(r.burntTokens.balances);
-        totalResult->nonConsortiumTokens.AddBalances(r.nonConsortiumTokens.balances);
-        totalResult->dexfeeburn.AddBalances(r.dexfeeburn.balances);
-        totalResult->paybackFee.AddBalances(r.paybackFee.balances);
+        totalResult->burntDFI += r->burntDFI;
+        totalResult->burntFee += r->burntFee;
+        totalResult->auctionFee += r->auctionFee;
+        totalResult->burntTokens.AddBalances(r->burntTokens.balances);
+        totalResult->nonConsortiumTokens.AddBalances(r->nonConsortiumTokens.balances);
+        totalResult->dexfeeburn.AddBalances(r->dexfeeburn.balances);
+        totalResult->paybackFee.AddBalances(r->paybackFee.balances);
     }
 
+    GetMemoizedResultCache().Set(request, {height, hash, *totalResult});
```

Comment:
```suggestion
    MemoizedResultCache().Set(request, {height, hash, *totalResult});
```
`MemoizedResultCache` instead of `GetMemoizedResultCache` lest we have `GetMemoizedResultCache().get`